docs/report/main.tex

```diff
@@ -1057,17 +1198,43 @@ \subsection{Algorithmic issues}
 %	PROBLEM 2
 %----------------------------------------------------------------------------------------
 
-\section{References and Bibliographies}
 
+\section{References and Bibliographies}
```

Comment:
We should probably convert this to a BibTex citation in a separate file, and have LaTeX auto populate the bibliography.

Reply:
See comments on line 162